source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -195,12 +210,12 @@
         this.period = data.Period;
         this.currentPosition= data.CurrentPosition || '';
         this.seniority = data.Seniority || '';
-        this.evaluators = data.Evaluators;
+        this.evaluators(data.Evaluators);
         this.evaluatorsString = ko.computed(function () {
-            return this.evaluators.toString().replace(/,/g, ', ');
+            return this.evaluators().toString().replace(/,/g, ', ');
```

Comment:
Yes, that was working, just we add parenthesis, because now it is an observable property.